sdk/provisioning/Azure.Provisioning/src/Azure.Provisioning.csproj

```diff
@@ -27,7 +27,6 @@
     <PackageReference Include="Azure.ResourceManager.ApplicationInsights" />
     <PackageReference Include="Azure.ResourceManager.OperationalInsights" />
     <PackageReference Include="Azure.ResourceManager.ManagedServiceIdentities" />
```

Comment:
What is the thought here? Is this dependency not needed or we are going to get it indirectly from some other way?

Reply:
it is pulled in transitively from the ARM libraries.

Reply:
Oh yes, Mgmt packages took dependency for serialization APIs.